installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc

```diff
@@ -0,0 +1,27 @@
+[id="preparing-to-install-on-bare-metal"]
+= Preparing for bare metal cluster installation
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-bare-metal
+
+toc::[]
+
+[id="preparing_preparing-to-install-on-bare-metal"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You have read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+
+[id="choosing-a-method-to-install-ocp-on-bare-metal"]
+== Choosing a method to install {product-title} on bare metal
+
+You can install OpenShift Container Platform on bare metal by using one of the following deployment methods.
+
+* **xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned cluster on bare metal]**: You can install {product-title} on bare metal by using installer provisioning.
```

Comment:
Sorry, I missed this one before if you would please update:
s/Installer provisioned cluster/Installer-provisioned cluster

Reply:
I wondered about that one but wasn't sure. Thank you!